src/App.js

```diff
@@ -6,48 +6,48 @@ import ContactPage from './pages/Contact';
 import TeamPage from './pages/Team';
 import MissingPage from './pages/Missing';
 import Menu from './components/Menu';
-import LangSelect from './components/LangSelect';
-import { NAVIGATION } from './lib/constants';
+import NavFooter from './components/Footer';
+import { NAVIGATION, LOCALES } from './lib/constants';
 
 function App() {
   const location = useLocation();
-  const { i18n } = useTranslation();
+  const { t, i18n } = useTranslation();
 
   useEffect(() => {
     const pagePath = location.pathname;
-    console.log("location changed!", pagePath.split("/"));
```

Comment:
Maybe remove your console.log statements after you are done debugging as it can slow down the site (there are a few throughout your code)

Reply:
good catch!